### ISSUE TITLE
AzureRM.Sql.Netcore - Get-AzureRmSqlDatabase  is not working.

### ISSUE THREAD

**Issue by MaximoTrinidad** [Azure/azure-powershell#6052]
### Description

The Get-AzureRmSqlDatabase cmdlet is not working when providing the required parameters.
ServerName, ResourceGroupName, and even with Databasename.

### Script/Steps for Reproduction

Execute the following command:
```powershell
Get-AzureRmSqlDatabase -ServerName azsqlseast01 -ResourceGroupName ArmPSCoreResources1 -DatabaseName "MySampleDatabase";
```
Gives the following error:
```
Get-AzureRmSqlDatabase : Unable to deserialize the response.                                  
At line:1 char:1                                                                              
+ Get-AzureRmSqlDatabase -ServerName azsqlseast01 -ResourceGroupName Ar ...                   
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                       
+ CategoryInfo          : CloseError: (:) [Get-AzureRmSqlDatabase], SerializationException    
+ FullyQualifiedErrorId : Microsoft.Azure.Commands.Sql.Database.Cmdlet.GetAzureSqlDatabase    
```
### Module Version

```powershell
                                                                                                     
PS /home/maxt> Get-Module -Listavailable AzureRm.Sql.Netcore | Select name, version                  
                                                                                                     
Name                Version                                                                          
----                -------                                                                          
AzureRM.Sql.Netcore 0.11.0            

```

### Environment Data

Ubuntu: 17.10
```
PS /home/maxt> cat /etc/os-release                     
NAME="Ubuntu"                                          
VERSION="16.04.4 LTS (Xenial Xerus)"                   
ID=ubuntu                                              
ID_LIKE=debian                                         
PRETTY_NAME="Ubuntu 16.04.4 LTS"                       
VERSION_ID="16.04"                                     
HOME_URL="http://www.ubuntu.com/"                      
SUPPORT_URL="http://help.ubuntu.com/"                  
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"     
VERSION_CODENAME=xenial                                
UBUNTU_CODENAME=xenial                                 
```
Azure Cloud Shell - Bash/pwsh
```
Azure:/
PS Azure:\> cat /etc/os-release
NAME="Ubuntu"
VERSION="16.04.4 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.4 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
VERSION_CODENAME=xenial
UBUNTU_CODENAME=xenial
Azure:/
PS Azure:\>
```
```powershell
PS /home/maxt> $psversiontable                                                                       
                                                                                                     
Name                           Value                                                                 
----                           -----                                                                 
PSVersion                      6.1.0-preview.1                                                       
PSEdition                      Core                                                                  
GitCommitId                    v6.1.0-preview.1                                                      
OS                             Linux 4.4.0-17134-Microsoft #1-Microsoft Tue Apr 10 18:04:00 PST 2018 
Platform                       Unix                                                                  
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                               
PSRemotingProtocolVersion      2.3                                                                   
SerializationVersion           1.1.0.1                                                               
WSManStackVersion              3.0                                                                   
```

### Debug Output

```
 Azure:\> $DebugPreference = 'Continue'
Azure:/
PS Azure:\> Get-AzureRmSqlDatabase -ServerName "azsqlseast01" -ResourceGroupName "ArmPSCoreResources1"
DEBUG: 11:20:53 PM - GetAzureSqlDatabase begin processing with ParameterSet '__AllParameterSets'.
DEBUG: 11:20:53 PM - using account id '30b10c7a-3cb6-4fed-9e3e-d63416cfd909'...
DEBUG: [Common.Authentication]: Authenticating using Account: '30b10c7a-3cb6-4fed-9e3e-d63416cfd909', environment: 'AzureCloud', tenant: '04b51827-7e6c-4caa-9f59-f599c5f59a00'
DEBUG: [HttpClientOperations]: Adding Header 'Metadata'
DEBUG: Checking Cache request http://localhost:50342/oauth2/token?resource=https:%2F%2Fmanagement.core.windows.net%2F&client_id=30b10c7a-3cb6-4fed-9e3e-d63416cfd909&api-version=2018-02-01
DEBUG: Cache Hit
DEBUG: ============================ HTTP REQUEST ============================

HTTP Method:
GET

Absolute Uri:
https://management.azure.com/subscriptions/606ba8bb-2f0e-4ebb-87d5-d5479cbffdee/resourceGroups/ArmPSCoreResources1/providers/Microsoft.Sql/servers/azsqlseast01/databases?api-version=2014-04-01

Headers:
x-ms-client-request-id        : ab919d9f-36f0-4c04-a9e8-00de3eb9b7e6
accept-language               : en-US

Body:



DEBUG: ============================ HTTP RESPONSE ============================

Status Code:
OK

Headers:
Cache-Control                 : no-store, no-cache
x-ms-request-id               : 3a5e7088-f99f-40ec-9bad-e4a05fada8cb
X-Content-Type-Options        : nosniff
DataServiceVersion            : 1.0;
Strict-Transport-Security     : max-age=31536000; includeSubDomains
Server                        : Microsoft-HTTPAPI/2.0
x-ms-ratelimit-remaining-subscription-reads: 14993
x-ms-correlation-request-id   : 78c7b2d4-1f86-4aad-b97e-cb44e4d90ef1
x-ms-routing-request-id       : EASTUS:20180425T232053Z:78c7b2d4-1f86-4aad-b97e-cb44e4d90ef1
Date                          : Wed, 25 Apr 2018 23:20:53 GMT

Body:
<feed xml:base="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/" xmlns="http://www.w3.org/2005/Atom" xmlns:d="http://schemas.microsoft.com/ado/2007/08/dataservices" xmlns:m="http://schemas.microsoft.com/ado/2007/08/dataservices/metadata">
  <id>https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/subscriptions/606ba8bb-2f0e-4ebb-87d5-d5479cbffdee/resourceGroups/ArmPSCoreResources1/providers/Microsoft.Sql/servers/azsqlseast01/databases</id>
  <title type="text">databases</title>
  <updated>2018-04-25T23:20:53Z</updated>
  <link rel="self" title="databases" href="databases" />
  <entry>
    <id>https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/databases/master</id>
    <category term="Microsoft.SqlServer.Management.Service.Domain.ArmResourceProvider.Database" scheme="http://schemas.microsoft.com/ado/2007/08/dataservices/scheme" />
    <link rel="edit" title="Database" href="databases/master" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/dataWarehouseQueries" type="application/atom+xml;type=feed" title="dataWarehouseQueries" href="databases/master/dataWarehouseQueries" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/usages" type="application/atom+xml;type=feed" title="usages" href="databases/master/usages" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/operationresults" type="application/atom+xml;type=feed" title="operationresults" href="databases/master/operationresults" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/azureAsyncOperation" type="application/atom+xml;type=feed" title="azureAsyncOperation" href="databases/master/azureAsyncOperation" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/importExportOperationResults" type="application/atom+xml;type=feed" title="importExportOperationResults" href="databases/master/importExportOperationResults" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/replicationLinks" type="application/atom+xml;type=feed" title="replicationLinks" href="databases/master/replicationLinks" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/extensions" type="application/atom+xml;type=feed" title="extensions" href="databases/master/extensions" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/auditingPolicies" type="application/atom+xml;type=feed" title="auditingPolicies" href="databases/master/auditingPolicies" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/connectionPolicies" type="application/atom+xml;type=feed" title="connectionPolicies" href="databases/master/connectionPolicies" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/securityMetrics" type="application/atom+xml;type=feed" title="securityMetrics" href="databases/master/securityMetrics" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/backupLongTermRetentionPolicies" type="application/atom+xml;type=feed" title="backupLongTermRetentionPolicies" href="databases/master/backupLongTermRetentionPolicies" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/geoBackupPolicies" type="application/atom+xml;type=feed" title="geoBackupPolicies" href="databases/master/geoBackupPolicies" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/queryStore" type="application/atom+xml;type=feed" title="queryStore" href="databases/master/queryStore" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/topQueries" type="application/atom+xml;type=feed" title="topQueries" href="databases/master/topQueries" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/dataMaskingPolicies" type="application/atom+xml;type=feed" title="dataMaskingPolicies" href="databases/master/dataMaskingPolicies" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/serviceTierAdvisors" type="application/atom+xml;type=feed" title="serviceTierAdvisors" href="databases/master/serviceTierAdvisors" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/upgradeHint" type="application/atom+xml;type=entry" title="upgradeHint" href="databases/master/upgradeHint" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/schemas" type="application/atom+xml;type=feed" title="schemas" href="databases/master/schemas" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/transparentDataEncryption" type="application/atom+xml;type=feed" title="transparentDataEncryption" href="databases/master/transparentDataEncryption" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/restorePoints" type="application/atom+xml;type=feed" title="restorePoints" href="databases/master/restorePoints" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/metrics" type="application/atom+xml;type=feed" title="metrics" href="databases/master/metrics" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/metricDefinitions" type="application/atom+xml;type=feed" title="metricDefinitions" href="databases/master/metricDefinitions" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/securityAlertPolicies" type="application/atom+xml;type=feed" title="securityAlertPolicies" href="databases/master/securityAlertPolicies" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/advisors" type="application/atom+xml;type=feed" title="advisors" href="databases/master/advisors" />
    <title />
    <updated>2018-04-25T23:20:53Z</updated>
    <author>
      <name />
    </author>
    <m:action metadata="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/$metadata#ArmResourceProviderModel.pause" title="pause" target="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/databases/master/pause" />
    <m:action metadata="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/$metadata#ArmResourceProviderModel.resume" title="resume" target="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/databases/master/resume" />
    <m:action metadata="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/$metadata#ArmResourceProviderModel.ActivateDatabase" title="ActivateDatabase" target="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/databases/master/ActivateDatabase" />
    <m:action metadata="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/$metadata#ArmResourceProviderModel.DeactivateDatabase" title="DeactivateDatabase" target="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/databases/master/DeactivateDatabase" />
    <m:action metadata="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/$metadata#ArmResourceProviderModel.export" title="export" target="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/databases/master/export" />
    <content type="application/xml">
      <m:properties>
        <d:id>/subscriptions/606ba8bb-2f0e-4ebb-87d5-d5479cbffdee/resourceGroups/ArmPSCoreResources1/providers/Microsoft.Sql/servers/azsqlseast01/databases/master</d:id>
        <d:name>master</d:name>
        <d:type>Microsoft.Sql/servers/databases</d:type>
        <d:location>East US</d:location>
        <d:kind>v12.0,system</d:kind>
        <d:managedBy>/subscriptions/606ba8bb-2f0e-4ebb-87d5-d5479cbffdee/resourceGroups/ArmPSCoreResources1/providers/Microsoft.Sql/servers/azsqlseast01</d:managedBy>
        <d:properties m:type="Microsoft.SqlServer.Management.Service.Domain.ArmResourceProvider.Database_DatabaseProperties">
          <d:databaseId m:type="Edm.Guid">123b774c-ebfd-4b67-bf07-658872f75407</d:databaseId>
          <d:edition>System</d:edition>
          <d:status>Online</d:status>
          <d:serviceLevelObjective>System0</d:serviceLevelObjective>
          <d:collation>SQL_Latin1_General_CP1_CI_AS</d:collation>
          <d:maxSizeBytes m:type="Edm.Int64">32212254720</d:maxSizeBytes>
          <d:creationDate m:type="Edm.DateTime">2018-04-25T22:13:41.517Z</d:creationDate>
          <d:currentServiceObjectiveId m:type="Edm.Guid">29dd7459-4a7c-4e56-be22-f0adda49440d</d:currentServiceObjectiveId>
          <d:requestedServiceObjectiveId m:type="Edm.Guid">29dd7459-4a7c-4e56-be22-f0adda49440d</d:requestedServiceObjectiveId>
          <d:requestedServiceObjectiveName>System0</d:requestedServiceObjectiveName>
          <d:createMode m:null="true" />
          <d:sampleName m:null="true" />
          <d:sourceDatabaseId m:null="true" />
          <d:defaultSecondaryLocation>West US</d:defaultSecondaryLocation>
          <d:earliestRestoreDate m:type="Edm.DateTime" m:null="true" />
          <d:restorePointInTime m:type="Edm.DateTime" m:null="true" />
          <d:sourceDatabaseDeletionDate m:type="Edm.DateTime" m:null="true" />
          <d:blobUriAndSasKey m:null="true" />
          <d:recoveryServicesRecoveryPointResourceId m:null="true" />
          <d:elasticPoolName m:null="true" />
          <d:containmentState m:type="Edm.Int32">2</d:containmentState>
          <d:readScale>Disabled</d:readScale>
          <d:failoverGroupId m:null="true" />
          <d:zoneRedundant m:type="Edm.Boolean">false</d:zoneRedundant>
          <d:isUpgradeRequested m:type="Edm.Boolean">false</d:isUpgradeRequested>
        </d:properties>
      </m:properties>
    </content>
  </entry>
  <entry>
    <id>https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/databases/MySampleDatabase</id>
    <category term="Microsoft.SqlServer.Management.Service.Domain.ArmResourceProvider.Database" scheme="http://schemas.microsoft.com/ado/2007/08/dataservices/scheme" />
    <link rel="edit" title="Database" href="databases/MySampleDatabase" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/dataWarehouseQueries" type="application/atom+xml;type=feed" title="dataWarehouseQueries" href="databases/MySampleDatabase/dataWarehouseQueries" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/usages" type="application/atom+xml;type=feed" title="usages" href="databases/MySampleDatabase/usages" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/operationresults" type="application/atom+xml;type=feed" title="operationresults" href="databases/MySampleDatabase/operationresults" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/azureAsyncOperation" type="application/atom+xml;type=feed" title="azureAsyncOperation" href="databases/MySampleDatabase/azureAsyncOperation" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/importExportOperationResults" type="application/atom+xml;type=feed" title="importExportOperationResults" href="databases/MySampleDatabase/importExportOperationResults" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/replicationLinks" type="application/atom+xml;type=feed" title="replicationLinks" href="databases/MySampleDatabase/replicationLinks" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/extensions" type="application/atom+xml;type=feed" title="extensions" href="databases/MySampleDatabase/extensions" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/auditingPolicies" type="application/atom+xml;type=feed" title="auditingPolicies" href="databases/MySampleDatabase/auditingPolicies" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/connectionPolicies" type="application/atom+xml;type=feed" title="connectionPolicies" href="databases/MySampleDatabase/connectionPolicies" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/securityMetrics" type="application/atom+xml;type=feed" title="securityMetrics" href="databases/MySampleDatabase/securityMetrics" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/backupLongTermRetentionPolicies" type="application/atom+xml;type=feed" title="backupLongTermRetentionPolicies" href="databases/MySampleDatabase/backupLongTermRetentionPolicies" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/geoBackupPolicies" type="application/atom+xml;type=feed" title="geoBackupPolicies" href="databases/MySampleDatabase/geoBackupPolicies" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/queryStore" type="application/atom+xml;type=feed" title="queryStore" href="databases/MySampleDatabase/queryStore" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/topQueries" type="application/atom+xml;type=feed" title="topQueries" href="databases/MySampleDatabase/topQueries" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/dataMaskingPolicies" type="application/atom+xml;type=feed" title="dataMaskingPolicies" href="databases/MySampleDatabase/dataMaskingPolicies" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/serviceTierAdvisors" type="application/atom+xml;type=feed" title="serviceTierAdvisors" href="databases/MySampleDatabase/serviceTierAdvisors" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/upgradeHint" type="application/atom+xml;type=entry" title="upgradeHint" href="databases/MySampleDatabase/upgradeHint" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/schemas" type="application/atom+xml;type=feed" title="schemas" href="databases/MySampleDatabase/schemas" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/transparentDataEncryption" type="application/atom+xml;type=feed" title="transparentDataEncryption" href="databases/MySampleDatabase/transparentDataEncryption" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/restorePoints" type="application/atom+xml;type=feed" title="restorePoints" href="databases/MySampleDatabase/restorePoints" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/metrics" type="application/atom+xml;type=feed" title="metrics" href="databases/MySampleDatabase/metrics" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/metricDefinitions" type="application/atom+xml;type=feed" title="metricDefinitions" href="databases/MySampleDatabase/metricDefinitions" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/securityAlertPolicies" type="application/atom+xml;type=feed" title="securityAlertPolicies" href="databases/MySampleDatabase/securityAlertPolicies" />
    <link rel="http://schemas.microsoft.com/ado/2007/08/dataservices/related/advisors" type="application/atom+xml;type=feed" title="advisors" href="databases/MySampleDatabase/advisors" />
    <title />
    <updated>2018-04-25T23:20:53Z</updated>
    <author>
      <name />
    </author>
    <m:action metadata="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/$metadata#ArmResourceProviderModel.pause" title="pause" target="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/databases/MySampleDatabase/pause" />
    <m:action metadata="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/$metadata#ArmResourceProviderModel.resume" title="resume" target="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/databases/MySampleDatabase/resume" />
    <m:action metadata="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/$metadata#ArmResourceProviderModel.ActivateDatabase" title="ActivateDatabase" target="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/databases/MySampleDatabase/ActivateDatabase" />
    <m:action metadata="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/$metadata#ArmResourceProviderModel.DeactivateDatabase" title="DeactivateDatabase" target="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/databases/MySampleDatabase/DeactivateDatabase" />
    <m:action metadata="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/$metadata#ArmResourceProviderModel.export" title="export" target="https://management.eastus.control.database.windows.net/v2/ManagementService.Trusted.svc/modules/ServerManagement/databases/MySampleDatabase/export" />
    <content type="application/xml">
      <m:properties>
        <d:id>/subscriptions/606ba8bb-2f0e-4ebb-87d5-d5479cbffdee/resourceGroups/ArmPSCoreResources1/providers/Microsoft.Sql/servers/azsqlseast01/databases/MySampleDatabase</d:id>
        <d:name>MySampleDatabase</d:name>
        <d:type>Microsoft.Sql/servers/databases</d:type>
        <d:location>East US</d:location>
        <d:kind>v12.0,user</d:kind>
        <d:managedBy m:null="true" />
        <d:properties m:type="Microsoft.SqlServer.Management.Service.Domain.ArmResourceProvider.Database_DatabaseProperties">
          <d:databaseId m:type="Edm.Guid">f9bac9cf-42b6-46c5-b307-13df7415f446</d:databaseId>
          <d:edition>Standard</d:edition>
          <d:status>Online</d:status>
          <d:serviceLevelObjective>S0</d:serviceLevelObjective>
          <d:collation>SQL_Latin1_General_CP1_CI_AS</d:collation>
          <d:maxSizeBytes m:type="Edm.Int64">268435456000</d:maxSizeBytes>
          <d:creationDate m:type="Edm.DateTime">2018-04-25T22:14:34.963Z</d:creationDate>
          <d:currentServiceObjectiveId m:type="Edm.Guid">f1173c43-91bd-4aaa-973c-54e79e15235b</d:currentServiceObjectiveId>
          <d:requestedServiceObjectiveId m:type="Edm.Guid">f1173c43-91bd-4aaa-973c-54e79e15235b</d:requestedServiceObjectiveId>
          <d:requestedServiceObjectiveName>S0</d:requestedServiceObjectiveName>
          <d:createMode m:null="true" />
          <d:sampleName m:null="true" />
          <d:sourceDatabaseId m:null="true" />
          <d:defaultSecondaryLocation>West US</d:defaultSecondaryLocation>
          <d:earliestRestoreDate m:type="Edm.DateTime">2018-04-25T22:45:25.877Z</d:earliestRestoreDate>
          <d:restorePointInTime m:type="Edm.DateTime" m:null="true" />
          <d:sourceDatabaseDeletionDate m:type="Edm.DateTime" m:null="true" />
          <d:blobUriAndSasKey m:null="true" />
          <d:recoveryServicesRecoveryPointResourceId m:null="true" />
          <d:elasticPoolName m:null="true" />
          <d:containmentState m:type="Edm.Int32">2</d:containmentState>
          <d:readScale>Disabled</d:readScale>
          <d:failoverGroupId m:null="true" />
          <d:zoneRedundant m:type="Edm.Boolean">false</d:zoneRedundant>
          <d:isUpgradeRequested m:type="Edm.Boolean">false</d:isUpgradeRequested>
        </d:properties>
      </m:properties>
    </content>
  </entry>
</feed>


Get-AzureRmSqlDatabase : Unable to deserialize the response.
At line:1 char:1
+ Get-AzureRmSqlDatabase -ServerName "azsqlseast01" -ResourceGroupName  ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : CloseError: (:) [Get-AzureRmSqlDatabase], SerializationException
+ FullyQualifiedErrorId : Microsoft.Azure.Commands.Sql.Database.Cmdlet.GetAzureSqlDatabase

DEBUG: AzureQoSEvent: CommandName - Get-AzureRmSqlDatabase; IsSuccess - False; Duration - 00:00:00.2259106; Exception - Microsoft.Rest.SerializationException: Unable to deserialize the response. ---> Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonTextReader.Read()
   at Newtonsoft.Json.JsonReader.ReadForType(JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Microsoft.Rest.Serialization.SafeJsonConvert.DeserializeObject[T](String json, JsonSerializerSettings settings)
   at Microsoft.Azure.Management.Sql.DatabasesOperations.<ListByServerWithHttpMessagesAsync>d__12.MoveNext()
   --- End of inner exception stack trace ---
   at Microsoft.Azure.Management.Sql.DatabasesOperations.<ListByServerWithHttpMessagesAsync>d__12.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Azure.Management.Sql.DatabasesOperationsExtensions.<ListByServerAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Azure.Management.Sql.DatabasesOperationsExtensions.ListByServer(IDatabasesOperations operations, String resourceGroupName, String serverName, String expand, String filter)
   at Microsoft.Azure.Commands.Sql.Database.Services.AzureSqlDatabaseCommunicator.List(String resourceGroupName, String serverName)
   at Microsoft.Azure.Commands.Sql.Database.Services.AzureSqlDatabaseAdapter.ListDatabases(String resourceGroupName, String serverName)
   at Microsoft.Azure.Commands.Sql.Database.Cmdlet.GetAzureSqlDatabase.GetEntity()
   at Microsoft.Azure.Commands.Sql.Common.AzureSqlCmdletBase`2.ExecuteCmdlet()
   at Microsoft.WindowsAzure.Commands.Utilities.Common.AzurePSCmdlet.ProcessRecord();
DEBUG: Finish sending metric.
DEBUG: 11:20:54 PM - GetAzureSqlDatabase end processing.
DEBUG: 11:20:54 PM - GetAzureSqlDatabase end processing.
```
![get-azurermsqldatabase_01_2018-04-25_18-58-33](https://user-images.githubusercontent.com/19892435/39277855-f000d5e4-48bd-11e8-816f-d91ca30446d9.jpg)

**Comment by MaximoTrinidad** [comment]
FYI - Of course! Works on both Windows PowerShell and PowerShell Core in Windows.
But not in Linux!!

😀 
![get-azurermsqldatabase_02_2018-04-25_18-58-33](https://user-images.githubusercontent.com/19892435/39278320-7d153ce8-48c0-11e8-9e4a-8be5c9e1374e.jpg)

**Comment by MiYanni** [comment]
Even on my test branch (for the 0.12.0 upcoming build), this issue still exists. I get similar errors when creating a database, too (but the database creation does succeed). 

```powershell
New-AzureRmSqlDatabase -DatabaseName miyanni-sqldb -ServerName miyanni-sqlserver2 -ResourceGroupName miyanni-netcore-test
```
![image](https://user-images.githubusercontent.com/17788297/39776240-0fc03c0c-52b5-11e8-98b3-b071e0d21c2a.png)

From the Get:

```powershell
Get-AzureRmSqlDatabase -DatabaseName miyanni-sqldb -ServerName miyanni-sqlserver2 -ResourceGroupName miyanni-netcore-test 
```
![image](https://user-images.githubusercontent.com/17788297/39778307-cac72cfe-52ba-11e8-9435-9891c60df7c9.png)

@akromm @jaredmoo Can you guys investigate this issue? It is NetCore only. This happens on the current version (0.11.0) and will also occur on 0.12.0.

**Comment by jaredmoo** [comment]
Thanks for the report! I believe #5949 should fix this.

**Comment by MaximoTrinidad** [comment]
Awesome guys!! I'm looking forward to the fix. 
:)

**Comment by MaximoTrinidad** [comment]
Thank You!!

**Comment by MiYanni** [comment]
@MaximoTrinidad @jaredmoo Fixed in [0.12.0 for NetCore](https://github.com/Azure/azure-powershell/releases/tag/v0.12.0-nc-June2018)

**Comment by MaximoTrinidad** [comment]
Awesome!! I will be checking it out.

Thanks!
:)
